content/docs/tasks/security/authn-policy/index.md

```diff
@@ -570,6 +570,11 @@ $ for i in `seq 1 10`; do curl --header "Authorization: Bearer $TOKEN" $INGRESS_
 401
 {{< /text >}}
 
+You can add a JWT policy on `istio-ingressgateway`. This is often used in the
```

Comment:
Simplify:

This is often used to define JWT policies that apply at the gateway, instead of for individual services.

Reply:
thanks, updated.